draft-ietf-tls-tls13.md

```diff
@@ -2183,8 +2183,10 @@ hidden by TCP.
 
 A small allowance for errors in clocks and variations in measurements
 is advisable.  However, any allowance also increases the opportunity
-for replay.  In this case, it is better to reject early data than to
-risk greater exposure to replay attacks.
+for replay.  In this case, it is better to reject early data and fall back
+to a full 1-RTT handshake than to risk greater exposure to replay attacks,
+so in common network topologies, a small allowance on the order of ten
+seconds is reasonable.
```

Comment:
That's helpful in giving us numbers, but I'm not 100% clear still.  Maybe I'll ask differently.

I'm interested in the drift rate, which sounds like it might in the range of 0-200 milliseconds per day for the bulk of your clients.  You keep using absolute times, which leads me to assume that you are looking at the window you are enforcing on your servers.  That doesn't tell me much about the statistics of actual drift rates, since many clients will return within much shorter times than the week.

Am I right in saying that you have tickets that are valid for a week, and you have the time set to 120s, which gives you a 0.1% 0-RTT rejection rate (similarly for 10s and 0.3%)?

I understand that you have your own constraints when choosing what numbers to pick, but I would suggest that we advise servers to pick a lower number (which results in a higher reject rate), simply because anything we say is an endorsement of something.  Endorsing a replay window of 4 minutes is probably not quite conservative enough for general advice.  If Facebook has a much better handle on their management of replays than in the general case, a larger window is reasonable for them.